hack/install-router.sh

```diff
@@ -13,8 +13,8 @@ OPENSHIFT="${3}"
 OS_ROOT=$(dirname "${BASH_SOURCE}")/..
 
 if [[ "${OPENSHIFT}" == "" ]]; then
-    if [[ "$(which osc)" != "" ]]; then
-        OPENSHIFT=$(which osc)
+    if [[ "$(which openshift)" != "" ]]; then
```

Comment:
It's worth discussing whether we should be calling 'openshift cli' or 'osc'.  The build will produce the symlinks for 'osc' now.